app/scripts/modules/core/src/application/nav/applicationNav.component.ts

```diff
@@ -37,7 +37,7 @@ export class ApplicationNavComponent implements IComponentOptions {
           analytics-event="{{dataSource.title}}"
           ng-class="{active: $ctrl.isActive(dataSource)}"
       >
-        <i ng-if="dataSource.icon" class="ds-icon fa fa-{{dataSource.icon}}"></i>
+        <i ng-if="dataSource.icon" class="{{dataSource.icon}}"></i>
```

Comment:
making the icon declaration the complete class, since new font awesome is a mix of `fa` and `fas` and `far`, which is fun

Reply:
this is a better approach anyway 👍

Reply:
💯